runtime/westend/src/lib.rs

```diff
@@ -1097,7 +1118,7 @@ construct_runtime! {
 		ElectionProviderMultiPhase: pallet_election_provider_multi_phase::{Pallet, Call, Storage, Event<T>, ValidateUnsigned} = 24,
 
 		// Provides a semi-sorted list of nominators for staking.
-		VoterList: pallet_bags_list::{Pallet, Call, Storage, Event<T>} = 25,
```

Comment:
Wow, I actually thought that this used to be called `BagsList`. If  it is called `VoterList`, let's just not migrate anything in Polkadot/Kusama/Westend. 

`VoterList`, and later on we add `TargetList`, which will be second instance of the bags-list. Sounds good?

Reply:
Assuming that making the pallet instanciable will not affect the storage prefix.

Reply:
Yeah I thought so too, but to decided to do it anyway as we have the migration to at least showcase it.